cypress_test/integration_tests/desktop/writer/focus_spec.js

```diff
@@ -15,7 +15,9 @@ describe('Focus tests', function() {
 
 	it('Basic document focus.', function() {
 		// Document has the focus after load
-		cy.document().its('activeElement.className')
+		cy.get('#coolframe')
```

Comment:
a lot of code repetition of these 3 statements, maybe there is a way to avoid it?